docs/kubernetes/how-to-setup-a-private-docker-registry-with-lke-and-object-storage/index.md

```diff
@@ -308,7 +308,7 @@ ingress:
     nginx.ingress.kubernetes.io/proxy-read-timeout: "6000"
     nginx.ingress.kubernetes.io/proxy-send-timeout: "6000"
   tls:
-    - secretName: docker-registry-prod
+    - secretName: letsencrypt-secret-prod
```

Comment:
This is incorrect based on my experience. This actually causes serious problems with cert-manager.

Reply:
This secret name is a reference to the certificate and not to the let's encrypt account secret.